addons/website_forum/views/website_forum.xml

```diff
@@ -379,23 +379,50 @@
 </template>
 
 <!-- All Forums Layout -->
-<template id="forum_all" name="Our Forums">
-    <t t-call="website.layout">
+<template id="forum_all_banner">
+  <section class="s_cover parallax s_parallax_is_fixed bg-black-50 pt64 pb64" data-scroll-background-ratio="1">
+    <span class="s_parallax_bg oe_img_bg oe_custom_bg" style="background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0;" />
         <div class="container">
-            <h1 class="mb32">Our forums</h1>
-            <div class='row'>
-                <div t-if="not forums" class="alert alert-info">No forum post yet.</div>
-                <div t-foreach="forums" t-as="forum" class="col-md-3 text-center mb32">
-                    <a t-attf-href="/forum/#{ slug(forum) }">
-                        <div class="fa fa-5x fa-comment mb16" role="img" aria-label="Forum" title="Forum"/>
-                        <div t-field="forum.name"/>
-                    </a>
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-12 s_title" data-name="Title">
+                    <h1 class="s_title_thin o_default_snippet_text text-center">Our forums</h1>
+                </div>
+                <div class="col-lg-12 s_text pt16 pb16" data-name="Text">
+                    <p class="lead o_default_snippet_text" style="text-align: center;">Write one or two paragraphs describing your product, services or a specific feature.
+                        <br /> To be successful your content needs to be useful to your readers.
+                    </p>
                 </div>
             </div>
         </div>
-    </t>
+    </section>
 </template>
 
+<template id="forum_all" name="Forum Navigation">
+    <t t-call="website.layout">
+        <t t-set="pageName" t-value="'forums'"/>
+        <div id="wrap" class="website_forum">
+            <t t-call="website_forum.forum_all_banner"/>
```

Comment:
This won't be editable the right way as it is not in an `oe_structure`... @JKE-be should we add an oe_structure and do an xpath on it so that the oe_structure "fork" system works ? Never tried that.